<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ICS-LT-BLK96V3" name="Default test(failed)" tests="2" failures="1" timestamp="5 Jan 2024 06:59:31 GMT" time="0.009" errors="0">
  <testcase name="test2" time="0.0" classname="com.infinite.priority.TestNGRetryFailDemo"/>
  <testcase name="test3" time="0.001" classname="com.infinite.priority.TestNGRetryFailDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.infinite.priority.TestNGRetryFailDemo.test3(TestNGRetryFailDemo.java:22)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
</testsuite> <!-- Default test(failed) -->
